### ISSUE TITLE
Since 1.0.0, `easing` no longer works

### ISSUE THREAD

**Issue by stowball** [oblador/react-native-animatable#78]
I noticed that my custom loading spinner didn't look right, and it appears that using the `easing` prop now has no effect.

The following code

```js
<Animatable.View
    animation="rotate"
    duration={1000}
    easing="linear"
    iterationCount="infinite"
    style={{
        backgroundColor: 'red',
        height: 50,
        width: 50
    }}
/>
```

will produce this (note the "pulsing" as it spins):

![animatable-easing-bug](https://cloud.githubusercontent.com/assets/853552/20734450/aa322f6c-b6ed-11e6-93ed-f4093864a462.gif)

But, if I change the fallback from `ease` to `linear` on https://github.com/oblador/react-native-animatable/blob/master/createAnimatableComponent.js#L312, then it produces a correct rotating animation:

![animatable-easing-fix](https://cloud.githubusercontent.com/assets/853552/20734474/dbb083ae-b6ed-11e6-84dd-b9e66d9f68ba.gif)

**Comment by oblador** [comment]
Fixed in [1.1.0](https://github.com/oblador/react-native-animatable/releases/tag/v1.1.0).

**Comment by stowball** [comment]
Thank you!
